tools/cli/commands/create.py

```diff
@@ -373,6 +388,11 @@ def flags(parser):
         dest='network_name',
         default=_DATALAB_NETWORK,
         help='name of the network to which the instance will be attached.')
+    parser.add_argument(
+        '--subnet-name',
+        dest='subnet_name',
+        default=_DATALAB_SUBNET,
```

Comment:
I would recommend not providing a default value, so that later on we can check whether or not any value was provided with the conditional:

```py
if args.subnet_name is not None:
    ...
```